src/components/MessageList/VirtualizedMessageList.js

```diff
@@ -34,6 +34,8 @@ const VirtualizedMessageList = ({
   loadingMore,
   messageLimit = 100,
   overscan = 200,
+  noGroupByUser = false,
```

Comment:
What you think of:
```suggestion
  shouldGroupByUser = true,
```
Just to avoid the double negation `!noGroupByUser` later

Reply:
I was going the same direction as MessageList component but this makes more sense, will update